src/core/migrator.js

```diff
@@ -67,7 +68,13 @@ export function getMigrator (type, args) {
         if (helpers.version.getDialectName() === 'pg') {
           const customSchemaName = helpers.umzug.getSchema('migration');
           if (customSchemaName && customSchemaName !== 'public') {
-            return sequelize.createSchema(customSchemaName);
+            return sequelize.createSchema(customSchemaName).catch(err => {
```

Comment:
This can be changed to only create schema if it doesn't exist, [`showAllSchemas`](https://github.com/sequelize/sequelize/blob/d43f162d75203ddadfc4a2479215d53204cc5c7b/lib/query-interface.js#L111) can be used